modular_skyrat/modules/blueshield/code/modules/clothing/head/jobs.dm

```diff
@@ -1,14 +1,15 @@
 /obj/item/clothing/head/beret/blueshield
-	name = "blueshield's beret" //Thank you for helping me rework the armor values for all Blueshield items Trilby <3 -CinderWC
-	desc = "Granted to those under the prestigious rank of blueshield... or just fancy bodyguard. Protects your head quite well, being made of industrial durathread fibers."
+	name = "blueshield's beret" //Thank you for helping me rework the armor values for all Blueshield items Trilby <3 -CinderWC  //Hi Alek here, I'm gonna fuck with the values. Sorry!
```

Comment:
```suggestion
	name = "blueshield's beret"
```
Dumbshit code comments like these were a scourge during the old codebase so do us a favour and rightfully nuke one from existence. Whoever ported Blueshields over must've not noticed this comment.